### PR TITLE
Accentuate hidden taxonomies

### PR DESCRIPTION
**Pull request by nikolas**
Because we're using micro-monads we might need to rethink how we do the
render layout. Anyways, this is a rather involved workaround that, for the
time being, introduces the concept of an "accentuated taxonomy" into the
file flippers -- to save on gas.

Imagine two roads:

```
------------------|
|
------------------|
------------------|
|
------------------|
|
------------------|
|
------------------|
|
------------------|
```

One going right and one going up. Our taxonomies CRM redefines the
bounding boxes, so it looks more like this now:

```
                __ooooooooo__
             oOOOOOOOOOOOOOOOOOOOOOo
         oOOOOOOOOOOOOOOOOOOOOOO OOOOOOo
      oOOOOOOOOO  OOOOOOOOOOOO  OOOOOOOOOOo
    oOOOOOOOOOOOOO  OOOOOOOOO  OOOOOOOOOOOOOo
  oOOOOOOOOOOOOOOOOO OOOOOOO OOOOOOOOOOOOOOOOOo
 oOOOOOOOOOOO*  *OOOO OOOOO OOO*  *OOOOOOOOOOOOo
oOOOOOOOOOOO      OOOOOOOOOOOO      OOOOOOOOOOOOo
oOOOOOOOOOOOOo  oOOOOOOOOOOOOOOo  oOOOOOOOOOOOOOo
oOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOo
oOOOO     OOOOOOOOOOOO  OO  OOOOOOOOOOOOO     OOOOo
oOOOOOO OOOOOOOOOOOOOOOO   OOOOOOOOOOOOOOOO OOOOOOo
 *OOOOO  OOOOOOOOOOOOOOO   OOOOOOOOOOOOOOO  OOOOO*
*OOOOOO  *OOOOOOOOOOOO  OO  OOOOOOOOOOO*  OOOOOO*
 *OOOOOO  *OOOOOOOOOOOOOOOOOOOOOOOOOOO*  OOOOOO*
  *OOOOOOo  *OOOOOOOOOOOOOOOOOOOOOOO*  oOOOOOO*
    *OOOOOOOo  *OOOOOOOOOOOOOOOOO*  oOOOOOOO*
      *OOOOOOOOo  *OOOOOOOOOOO*  oOOOOOOOO*      A heart at peace
         *OOOOOOOOo           oOOOOOOOO*      gives life to the body.
             *OOOOOOOOOOOOOOOOOOOOO*                    - Proverbs 14:30
                  ""ooooooooo""
```

### CODE REVIEW

**Comment by nikolas** [comment]
whatever nevermind

**Comment by mcfiredrill** [comment]
wait

File: app/models/Drifter.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import systam
+
+class Driver
+  """
+Drivers don't have keys
```
Comment (mcfiredrill) [inline app/models/Drifter.rb:5]: screenshot 2015-06-23 10 40 17

**Comment by mcfiredrill** [comment]
Kind of worried about the dark emacs devices buffer overflowing into the vimball directory. 

Anything we can do about that?! :sweat:

**Comment by nikolas** [comment]
drivers wanted

**Comment by nikolas** [comment]
Closing to re-open on the 8th day of Bellbottom, Dewgong time.

**Pull request closed without merging**